apps/client/app/navbar.tsx

```diff
@@ -27,10 +29,27 @@ function getActiveNavItem(path: string): NavItem | undefined {
 }
 
 export default function Navbar() {
-  const {user, signOut} = useAuth();
+  const {user, signIn, signOut} = useAuth();
   const pathname = usePathname();
   const activeNavItem = getActiveNavItem(pathname);
 
+  const userFirstLetter = user ? user.email[0] : null;
+
+  const refreshToken = async () => {
+    const res = await AuthService.refreshToken();
+    Result.fold(
+      res,
+      (token) => {
+        if (token) {
+          signIn(token);
+          alert('Your token has been refreshed');
+        }
+      },
+      // in a real application we'd show an error
+      (err) => console.error(err),
+    );
+  };
```

Comment:
Just a demo feature, but it allows us to refresh the token from the UI when authenicated.